battleship.py

```diff
@@ -89,7 +115,17 @@ def checkShip(grid, ship):
 Returns: 2D list of ints
 '''
 def addShips(grid, numShips):
-    return
+    current = 0
+    while current < numShips:
+        ship = createShip()
+        bool = checkShip(grid,ship)
+        if bool == True:

```

Comment:
No need of checking equality with True in case of boolean.

Reply:
@Pragya28 I made changes in the function. Now you can check.